leveldbjni/src/main/native-package/configure.ac

```diff
@@ -0,0 +1,67 @@
+# ---------------------------------------------------------------------------
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+# 
+# http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ---------------------------------------------------------------------------
+
+## --------------------------------
+## Initialization macros.
+## --------------------------------
+AC_PREREQ([2.61])
```

Comment:
Could you explain why we need to add this file to the repo?

Reply:
Actually, all the changes in `leveldbjni/src/main/native-package` are auto-generated by autotools.  It was incorrectly packed with this pull request. I was fixing a build error caused by version mismatch of automake by using this file. It was an error on my end, the changes in  `leveldbjni/src/main/native-package` are unnecessary and I will remove them.